### ISSUE TITLE
[QA] Implement Tab Key Accessibility for Cookie Editing

### ISSUE THREAD

**Issue by kjnanda** [GoogleChromeLabs/ps-analysis-tool#299]
Tab key accessibility for cookie modification is currently not working. This functionality should be implemented to enhance user experience.

https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/15173772/1a6992fd-55c5-42c4-b825-0e80bad20c2d

**Comment by amovar18** [comment]
Closing this since the feature itself has been blocked.
